cypress/e2e/calculator.cy.js

```diff
@@ -5,218 +5,158 @@ describe('계산기 테스트', () => {
     cy.visit('http://127.0.0.1:5500/index.html');
   });
 
+  const calculation = (firstNumber, operation, secondNumber, expectedTotal) => {
+    cy.clickDigit(firstNumber);
+    cy.clickOperation(operation);
+    cy.clickDigit(secondNumber);
+    cy.clickOperation(OPERATION.EQUAL);
+    cy.getTotal(expectedTotal);
+  };
```

Comment:
👍👍👍

Reply:
여러 개의 인자를 받는 함수의 경우 사용하는 곳에서 쉽게 사용하고, 실수를 방지하기 위해 객체로 묶어서 인자를 받도록 변경해보셔도 좋을거 같네요!

```suggestion
  const calculation = ({ firstNumber, operation, secondNumber, expectedTotal }) => {
    cy.clickDigit(firstNumber);
    cy.clickOperation(operation);
    cy.clickDigit(secondNumber);
    cy.clickOperation(OPERATION.EQUAL);
    cy.getTotal(expectedTotal);
  };
```